ablog/commands.py

```diff
@@ -403,6 +403,12 @@ def ablog_post(filename, title=None, **kwargs):
     type=str,
     help="environment variable name storing GitHub access token",
 )
+@arg(
+    "--github-ssh",
+    dest="github_is_http",
+    action="store_true",
+    help="use ssh when cloning website",
```

Comment:
```suggestion
    default=False,
    help="use ssh when cloning website",
```
Maybe?

Reply:
🤔 I think it is more action="store_false"; I'll take another look at it.

Reply:
Could be both settings? Otherwise, this is good to merge.